arviz/stats/density_utils.py

```diff
@@ -522,6 +522,8 @@ def kde(x, circular=False, **kwargs):
     arviz.stats.density_utils.kde: Arviz KDE estimator
     """
     if circular:
+        if circular == "degrees":
```

Comment:
Given that `_kde_circular` assumes inputs are in radians I added this internal conversion. We can discuss where else we can place it or if it shoudl exist at all.